doc/src/how-to/vscodeext-how-to-debug-apps-qml.qdoc

```diff
@@ -14,12 +14,79 @@
     the QML debugger and the debugger for your operating system. Usually, you use
     GDB on Linux and \macos or the Visual Studio Windows debugger on Windows.
 
+    When using the QML debugger, you can:
+
+    \list
+        \li Show variables including the \e {This} variable
+        \li Manipulate values of variables
+        \li Add watch expressions
+        \li Add conditional breakpoints
+        \li Add expression evaluation
+    \endlist
+
     \note Debugging requires opening a socket at a TCP port, which presents a
-    security risk. Anyone on the Internet could connect to the application that
+    security risk. Anyone on the internet could connect to the application that
     you are debugging and execute any JavaScript functions. Therefore, you must
     make sure that the port is properly protected by a firewall.
 
-    \section1 Debug QML code
+    \section1 Launch and debug applications
+
+    To debug only the QML code in an application, you can first start the
+    application and then attach the QML debugger to it.
+
+    To launch and debug Qt Quick applications:
+
+    \list 1
+        \li Create a \c {launch.json} file for the project. For more information,
+            see \l {Debug Qt applications}.
+        \li Select \uicontrol {QML debugger}.
+        \li Select the \uicontrol {Qt: QML: Launch} debug configuration.
+    \endlist
+
+    The launch debug configuration adds the following lines to the
+    \c {launch.json} file:
+
+   \badcode
+   "configurations": [
+       {
+           "name": "Qt: QML: Launch",
+           "type": "qml",
+           "request": "launch",
+           "program": "${command:cmake.launchTargetPath}"
+       },
+       {
+           "type": "qml",
+           "request": "attach",
+           "name": "Attach to QML debugger",
+           "host": "localhost",
+           "port": "^\"Custom port or \\${command:qt-qml.debugPort} for compound usage\""
+       }
+   ]
+   \endcode
+
+    \section2 Set build folders
```

Comment:
`buildFolders` is a general feature available for both launch and attach modes. Can we also state that in the attach mode?